### ISSUE TITLE
Opening angle-bracket for generic type mistakenly parsed as less-than?

### ISSUE THREAD

**Issue by aliak00** [nicklockwood/SwiftFormat#325]
So this:

```diff
- var executeCallData: SynchronizedArray<(Error?) -> Void> = []
+ var executeCallData: SynchronizedArray < (Error?) -> Void> = []
```

Similar to issue #2 I guess.

**Comment by nicklockwood** [comment]
@aliak00 I’m guessing it’s the -> that’s confusing it - I don’t have a test for that case. Thanks for reporting, I’ll fix it asap.

**Comment by nicklockwood** [comment]
@aliak00 fixed in version 0.35.9
